airflow/api_fastapi/core_api/routes/ui/backfills.py

```diff
@@ -42,6 +42,7 @@
 
 @backfills_router.get(
     path="",
+    include_in_schema=False,
```

Comment:
@pierrejeambrun Should we instead just add this to the following line?

https://github.com/apache/airflow/blob/b7df4639b5b61cca0b5bac7687f1d6e6e4c6c40a/airflow/api_fastapi/core_api/routes/ui/__init__.py#L27

Reply:
Yes that's a better, updated.